i18n/en.json

```diff
@@ -447,6 +448,68 @@
   "admin.email.smtpUsernameExample": "E.g.: \"admin@yourcompany.com\", \"AKIADTOVBGERKLCBV\"",
   "admin.email.smtpUsernameTitle": "SMTP Server Username:",
   "admin.email.testing": "Testing...",
+  "admin.experimental.allowCustomThemes.desc": "Enables the **Display > Theme > Custom Theme** section in Account Settings.",
```

Comment:
This was super painful to author, even with the `mmjstool`. I'd love to spend some mana changing the `t` function to take the default English value as a second parameter and updating `mmjstool` to extract the default value.

Reply:
I was actually thinking about that recently. Perhaps we just get rid of `en.json` entirely and just have it solely based off of the default values. We discussed removing the default values in the past because they were redundant, but we liked seeing those strings in context, so perhaps `en.json` is the place that's redundant